R/writeComps.R

```diff
@@ -343,13 +336,7 @@ writeComps = function(inComps, fname = NULL, abins = NULL, lbins = NULL,
 
   # Now assemble everything and write to a file
   # Note that we are stripping the last, dummy bin.
-
   NCOLS <- ifelse(AAL, NCOLS - 5, NCOLS - 3)
-  # KFJ(2015-06-16): Can also use the following b/c
-  # with Inf as the last dummy column it will not be
-  # selected as a number, and if other columns are later
-  # added for some reason then the code will not break.
-  # NCOLS <- max(grep("[A-Z]{1}[0-9]+", colnames(output)))
 
   # CRW: I think this is creating a matrix of a specific size (mComps)
```

Comment:
Clean up this comment too (e.g. change to `# Create an empty matrix of size matching mComps`)?

Reply:
Cleaned up in commit 8693650cd6e3dbdeac3045f1ca7cea478f2cab8f